examples/corrr/app.R

```diff
@@ -38,9 +37,10 @@ server <- function(input, output, session) {
   output$plot <- renderPlot({
 
     df() %...>% {
-      dplyr::select(., tidyselect:::where(is.numeric)) %>%
-        corrr::correlate(x = ., quiet = TRUE) %>%
-        corrr::rplot() +
+
+      numeric_cols <- names(which(vapply(., FUN.VALUE = logical(1), FUN = is.numeric)))
```

Comment:
What happens if you leave of `names` and `which`?

Reply:
Works well! Sorry I missed that.